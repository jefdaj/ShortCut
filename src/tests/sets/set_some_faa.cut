# Basic test of the some function and its binary operator equivalent, ?.
# These need to be tested separately with literals and path-based types
# because they're implemented separately.

set1 = ["data/7942genes.faa", "data/7942genes10.faa", "data/7942genes50.faa"]
set2 = ["data/7942genes.faa", "data/7942genes10.faa"]

res1 = some [set1, set2]
res2 = set1 ? set2

result = [res1, res2]
