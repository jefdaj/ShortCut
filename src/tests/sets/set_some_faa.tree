.
|-- cache
|   `-- lines
|       |-- 4dc1af43cb.txt
|       |-- 6785e2593e.txt
|       |-- 690652ab83.txt
|       |-- 954a24e2cf.txt
|       |-- bdc783547b.txt
|       |-- ef87d3e725.txt
|       `-- f041cf50cb.txt
|-- exprs
|   |-- list
|   |   |-- 6fbe36f9fa.str.list -> ../../cache/lines/4dc1af43cb.txt
|   |   |-- b883840e2c.str.list -> ../../cache/lines/6785e2593e.txt
|   |   |-- c974d30b2b.str.list.list -> ../../cache/lines/f041cf50cb.txt
|   |   `-- efd1761a9f.str.list.list -> ../../cache/lines/690652ab83.txt
|   |-- some
|   |   `-- 2305ba347a.str.list -> ../../cache/lines/954a24e2cf.txt
|   `-- str
|       |-- 1d8a61c770.str -> ../../cache/lines/bdc783547b.txt
|       |-- cb201cf4f8.str -> ../../cache/lines/954a24e2cf.txt
|       `-- db90f32127.str -> ../../cache/lines/ef87d3e725.txt
`-- vars
    |-- res1.str.list -> ../exprs/some/2305ba347a.str.list
    |-- res2.str.list -> ../exprs/some/2305ba347a.str.list
    |-- result -> ../exprs/list/c974d30b2b.str.list.list
    |-- set1.str.list -> ../exprs/list/b883840e2c.str.list
    `-- set2.str.list -> ../exprs/list/6fbe36f9fa.str.list

7 directories, 20 files
