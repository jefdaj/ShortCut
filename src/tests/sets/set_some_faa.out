[["data/7942genes50.faa"], ["data/7942genes50.faa"]]
